boards/arm/stm32/nucleo-f4x1re/src/stm32_spi.c

```diff
@@ -128,6 +137,13 @@ void stm32_spi1select(FAR struct spi_dev_s *dev, uint32_t devid,
   spiinfo("devid: %d CS: %s\n", (int)devid, selected ? "assert" :
           "de-assert");
 
+#if defined(CONFIG_LCD_SSD1306_SPI)
+  if (devid == SPIDEV_DISPLAY(0))
```

Comment:
Optional
IMO this is exactly a right time to change `if`s to `switch`/`case` for `stm32_spi1select`.